frontend/portals/SearchSuggestions/index.jsx

```diff
@@ -0,0 +1,124 @@
+import React, {
+  memo, useEffect, useMemo, useRef,
+} from 'react';
```

Comment:
shouldn't the search suggestions be scrollable?

Reply:
Hm. Sure. They are scrollable on Android. Isnt it on iOs?

Reply:
tested on desktop chrome. couldn't also find any css that should do the horizontal scrolling of the word suggestions